flexget/plugins/input/regexp_parse.py

```diff
@@ -152,13 +154,19 @@ def isvalid(self, entry):
     @plugin.internet(log)
     def on_task_input(self, task, config):
         url = config['source']
+        encoding = config.get('encoding')
```

Comment:
You can just add the default here `config.get('encoding', 'utf-8')`. I'm not sure why we assume utf-8, but let's not break anything.

Reply:
Done as suggested. Thanks!